inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -463,7 +464,11 @@ study_data_cleaned <- study_data_cleaned %>%
                 period_end   = "recall_end",
                 datecol      = "enddate",
                 datereason   = "endcause" 
-               )
+               ) %>%
+  # label those that were present at the start/end
+  mutate(startcause = if_else(startdate == recall_start, "Present at start", startcause)) %>%
```

Comment:
ah didnt realise you meant in the script - coouldve just done this myself rather than bother you!! sorry